src/DocAssembler/DocAssembler/Actions/ConfigInitAction.cs

```diff
@@ -0,0 +1,104 @@
+﻿// <copyright file="ConfigInitAction.cs" company="DocFx Companion Tools">
+// Copyright (c) DocFx Companion Tools. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
+using DocAssembler.Configuration;
+using DocAssembler.FileService;
+using DocAssembler.Utils;
+using Microsoft.Extensions.Logging;
+
+namespace DocAssembler.Actions;
+
+/// <summary>
+/// Initialize and save an initial configuration file if it doesn't exist yet.
+/// </summary>
+public class ConfigInitAction
+{
+    private const string CONFIGFILENAME = ".docassembler.json";
```

Comment:
constansts are just PascalCase so, should be ConfigFileName

Reply:
Const must be in capitals (says the linter). No underscores or dashes allowed.

Reply:
Tried it again and you're right. Changed it.